### PR TITLE
Ledc clusters

### PR DESCRIPTION
**Pull request by burrbull**

### CODE REVIEW

**Comment by burrbull** [comment]
No more `paste` in LEDC.

**Review by MabezDev — APPROVED** [review]
Example still works. Really nice to remove paste macros, thanks!

**Comment by MabezDev** [comment]
Wait, does this PR not depend on https://github.com/esp-rs/esp-pacs/pull/223? Or will that be a follow up?

**Comment by burrbull** [comment]
> Wait, does this PR not depend on [esp-rs/esp-pacs#223](https://github.com/esp-rs/esp-pacs/pull/223)? Or will that be a follow up?

Depends. I thought I'd specified this.

**Comment by MabezDev** [comment]
Weird, not sure how its working, the rev specified is on your repo, but in `Cargo.toml` the repo points to esp-pacs :thinking: . Could you update the rev to point at the latest esp-pacs main and we'll get this merged :).

**Comment by burrbull** [comment]
> Weird, not sure how its working, the rev specified is on your repo

Yeah. I've found this several days ago. But it works not always.

**Review by MabezDev — APPROVED** [review]
Thanks!

File: esp-hal/CHANGELOG.md [hunk 1]
```diff
@@ -47,6 +47,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
   MCPWM clusters (#1360)
   TIMG clusters (#1364)
   SOC_ETM clusters (#1365)
+  LEDC clusters (#1368)
```
Comment (jessebraham) [inline esp-hal/CHANGELOG.md:50]: What is the reason for adding `CHANGELOG` entries for all of these PAC changes? This has absolutely zero impact on the end user as far as I can tell, and just appears to be implementation details. This just adds visual noise and does not present any useful information, IMO.

Comment (MabezDev) [reply esp-hal/CHANGELOG.md:50]: Fair point, I will remove them in my other PR and from now on add skip-changelog.

File: esp-hal/CHANGELOG.md [hunk 2]
```diff
@@ -45,9 +45,6 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - UART: Rework `change_baud` so it is possible to set baud rate even after instantiation (#1350)
 - Runtime ISR binding for SHA,ECC and RSA (#1354)
 - Update `pac`s with removed suffixes in `int` field names. Use `W1TC` for `int_clr` (#1357)
```
Comment (MabezDev) [inline esp-hal/CHANGELOG.md:47]:
```suggestion
```

We may as well remove the whole entry

**Pull request merged**